Tests/LibWeb/Layout/expected/table/nested-table-box-width.txt

```diff
@@ -84,23 +84,23 @@ ViewportPaintable (Viewport<#document>) [0,0 800x600]
       PaintableWithLines (BlockContainer(anonymous)) [8,8 784x0]
       PaintableWithLines (TableWrapper(anonymous)) [8,8 115.828125x124]
         PaintableBox (Box<TABLE>) [8,8 115.828125x124]
-          PaintableBox (Box<TBODY>) [13,13 99.828125x108] overflow: [13,13 103.828125x112]
-            PaintableBox (Box<TR>) [15,15 99.828125x54] overflow: [15,15 101.828125x110]
+          PaintableBox (Box<TBODY>) [15,15 101.828125x110]
+            PaintableBox (Box<TR>) [15,15 101.828125x54] overflow: [15,15 101.828125x110]
```

Comment:
Also here

Reply:
There's overflow here because of the `rowspan="2"` 

![image](https://github.com/user-attachments/assets/46d67074-c94f-4698-b0d4-df8110705d5c)